app/models/messages/Consignment.scala

```diff
@@ -28,6 +28,11 @@ case class Consignment(
   PlaceOfLoading: Option[PlaceOfLoading],
   HouseConsignment: Seq[HouseConsignment]
 ) {
+
+  def isTransportDefined: Option[Boolean] = modeOfTransportAtTheBorder match {
+    case Some(value) => Some(true)
```

Comment:
Could this be simplified to modeOfTransportAtBorder.isDefined with it giving a Boolean instead of a Option[Boolean]?

Reply:
Have used isDefined rather than case statement but left it as an option Boolean otherwise there are other implications